src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -27,6 +27,8 @@
 public class ParserUtil {
 
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
+    public static final String MESSAGE_INVALID_DOLLAR_AMOUNT =
+            "Dollar amount is not a non-zero number with at most 2 decimal places.";
```

Comment:
Might be quite confusing to read. Maybe rephrase to something like:
Invalid dollar amount. Dollar amount should be a non-zero number with at most 2 decimal places.

Reply:
Oops, it's a typo, my bad